tests/suites/test_suite_pkwrite.function

```diff
@@ -124,3 +125,77 @@ void pk_write_key_check(char *key_file, int is_der)
     goto exit; /* make the compiler happy */
 }
 /* END_CASE */
+
+/* BEGIN_CASE */
+void pk_write_check_public_key_derivation(char *priv_key_file,
```

Comment:
Minor naming nit: when I read `key_derivation` I automatically think about [this kind](https://en.wikipedia.org/wiki/Key_derivation_function) of key derivation, which this isn't. I'd like to suggest something like `pk_write_public_from_private` (or similar) to make things crystal clear and avoid unwanted brain reflexes :)